tests/environment_linux_amd64_freeze.yaml

```diff
@@ -0,0 +1,21 @@
+# For more details on the format and content:
+# https://github.com/robocorp/rcc/blob/master/docs/recipes.md#what-is-in-condayaml
+# Tip: Adding a link to the release notes of the packages helps maintenance and security.
+
+channels:
+  - conda-forge
+
+dependencies:
+  # Define conda-forge packages here -> https://anaconda.org/search
+  # When available, prefer the conda-forge packages over pip as installations are more efficient.
+  - python=3.9.13               # https://pyreadiness.org/3.9/ 
+  - pip=22.1.2                  # https://pip.pypa.io/en/stable/news/
+  - pip:
+      # Define pip packages here -> https://pypi.org/
+      - robotframework==6.0.2
+      - selenium==4.8.2
```

Comment:
I think selenium library for robotframework is enough.  I do not install it in docker container and it works

Reply:
If I remember correctly selenium will be installed anyway for robotframework-selenium_library to work.

Reply:
![obraz](https://github.com/Mobica/innoextract-wasm/assets/146205703/ca39cd87-6ff2-4456-aed5-ef5ddb2f2e93)